### DISCUSSION TITLE
Contributions wanted! We need more pretrained ControlLoRA models!!!

### DISCUSSION THREAD

**Discussion by HighCWu** [HighCWu/ControlLoRA#1] [Announcements]
I don't have enough datasets to create pretrained models like ControlNet. So I need your contributions.

**Comment by toyxyz** [comment]
What kind of dataset do you need? I would be grateful if you could show me a sample.

**Reply by HighCWu** [reply]
Thank you. I think a high quality pose dataset with prompts would be okay. My pose model only training with 100 images. For other type datasets, I could generate from diffusiondb.
